packages/bot-web-ui/src/components/transaction.jsx

```diff
@@ -182,7 +182,6 @@ const Transaction = ({ active_transaction_id, contract, setActiveTransactionId }
             </div>
             <div className='transactions__cell transactions__stake'>
                 <TransactionIconWithText
-                    icon={<Icon icon='IcContractBuyPrice' />}
```

Comment:
I think this icon had the next line `title={localize('Buy price')}` as its tooltip, should it now be over the message?

Reply:
Ah yes, we don't have to use the `TransactionIconWithText` component for displaying `Buy Price` in that case.

Reply:
This line of code generates the extra icon which was told in the card. Since the Money component which is used in the message already has dollar sign and the icon is not required I omitted that.

Reply:
Now I see nijil comment, I think that I should remove the TransactionIconWithText component and only use Money one hear.